libs/authorizenet/src/drivers/interfaces/authorize-net-service.interface.ts

```diff
@@ -1,8 +1,7 @@
 import { Observable } from 'rxjs';
 
-import { DaffAuthorizeNetTokenResponse } from '../../models/response/authorize-net-token-response';
 import { DaffAuthorizeNetTokenRequest } from '../../models/request/authorize-net-token-request';
 
-export interface DaffAuthorizeNetService<T extends DaffAuthorizeNetTokenRequest, V extends DaffAuthorizeNetTokenResponse> {
-	generateToken(paymentTokenRequest: T): Observable<V>;
+export interface DaffAuthorizeNetService<T extends DaffAuthorizeNetTokenRequest = DaffAuthorizeNetTokenRequest> {
+	generateToken(paymentTokenRequest: T): Observable<any>;
```

Comment:
I don't think so. This is different than most packages, because it needs to add a payment method object to the cart that is already compatible with the platform. Otherwise, you'd need an authorizeNet transformer in the cart package. Usually we can just make things into a predictable daff object that we then transform into a platform object via a transformer, but here, to keep payment specific transformers out of the cart, it must be an `any`. Unless we do something like have an injectable transformer into the cart package maybe? It would still be a really generic object though, because the payment objects are so different between payment methods and platforms.

Reply:
that makes sense

Reply:
actually couldn't you still type this in the authorizenet package while leaving it as `any` in the cart package?

Reply:
probably not, because there's no guarantee that a shopify authorizeNet payment object will even look like
```
code: string;
authorize_net_js: any;
```
It could be completely different